### PR TITLE
Prefix JS global vars.

### PR DESCRIPTION
**Pull request by juanfra**
**Fixes:** https://github.com/WordPress/twentytwenty/issues/131
**WP Username:** juanfra

### CODE REVIEW

File: assets/js/construct.js [hunk 1]
```diff
@@ -486,12 +486,12 @@ twentytwenty.smoothScroll = {
...
 					// Get options
-					var additionalOffset 	= $( this ).data( 'additional-offset' ),
-						scrollSpeed 		= $( this ).data( 'scroll-speed' ) ? $( this ).data( 'scroll-speed' ) : 500;
+					var additionalOffset = $( this ).data( 'additional-offset' ),
+					    scrollSpeed      = $( this ).data( 'scroll-speed' ) ? $( this ).data( 'scroll-speed' ) : 500;
 
 					// Determine offset
 					var originalOffset = $target.offset().top,
-						scrollOffset = additionalOffset ? originalOffset + additionalOffset : originalOffset;
+					    scrollOffset   = additionalOffset ? originalOffset + additionalOffset : originalOffset;
```
Comment (ntwb) [inline assets/js/construct.js]: The 4 spaces before `scrollOffset` should be indented with a _tab_

File: assets/js/construct.js [hunk 2]
```diff
@@ -486,12 +486,12 @@ twentytwenty.smoothScroll = {
 					event.preventDefault();
 
 					// Get options
-					var additionalOffset 	= $( this ).data( 'additional-offset' ),
-						scrollSpeed 		= $( this ).data( 'scroll-speed' ) ? $( this ).data( 'scroll-speed' ) : 500;
+					var additionalOffset = $( this ).data( 'additional-offset' ),
+					    scrollSpeed      = $( this ).data( 'scroll-speed' ) ? $( this ).data( 'scroll-speed' ) : 500;
```
Comment (ntwb) [inline assets/js/construct.js]: The 4 spaces before `scrollSpeed` should be indented with a _tab_

File: assets/js/construct.js [hunk 3]
```diff
@@ -12,8 +12,8 @@ var twentytwenty = twentytwenty || {},
 
 var $doc = $( document ),
     $win = $( window ),
-    winHeight = $win.height(),
-    winWidth = $win.width();
+    twentytwentyWinHeight = $win.height(),
+    twentytwentyWinWidth = $win.width();
```
Comment (ntwb) [inline assets/js/construct.js]: _Tabs_ not _spaces_ for indentation

**Comment by acosmin** [comment]
@ntwb @juanfra these issues are solved in this pull req - #101

I did close the issue as it's no longer needed.

**Comment by ntwb** [comment]
Closing this PR per above comment and issue commetn

**Pull request closed without merging**